### PR TITLE
Math Magicians project setup

### PR DESCRIPTION
**Pull request by virag-ky**
- set up the project with Create React App (CRA)
- added linters
- added README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,52 @@
...
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
-
-### Deployment
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
-
-### `npm run build` fails to minify
-
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./MIT.md) licensed.
```
Comment (zieeco) [inline README.md:52]: **[REQUIRED CHANGES]**

- Please, kindly update the MIT license link as right now, the link is broken. You use the MIT from the [microverse readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) and add it to the root directory of your project to make your readme appear more professional.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,52 @@
...
+Feel free to check the [issues page](../../issues/).
 
-## Learn More
+## Show your support
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+Give a ⭐️ if you like this project!
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+## 📝 License
```
Comment (zieeco) [inline README.md:51]: **[REQUIRED CHANGES]**

- Please, kindly add the "Acknowledgment" header before the `License` just after the `Show your support to make your readme consistent following the [microverse readme template structure](https://github.com/microverseinc/readme-template/blob/master/README.md) to make your readme look more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi, @virag-ky, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No Linter error :heavy_check_mark: 
- Project setup correctly :+1: 
- Descriptive PR :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DammyShittu — APPROVED** [review]
Hi @virag-ky,

## APPROVED 🟢

Great implementation of all the requirements in this milestone. 👏 

✔️ Good documentation

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**